treeherder/services/pulse/connection.py

```diff
@@ -0,0 +1,23 @@
+import environ
+from django.core.exceptions import ImproperlyConfigured
+from kombu import Connection
+
+env = environ.Env()
+
+# Used to specify the PulseGuardian account that will be used to create
+# ingestion queues for the exchanges specified in ``PULSE_DATA_INGESTION_SOURCES``.
+# See https://pulse.mozilla.org/whats_pulse for more info.
+# Example: "amqp://myuserid:mypassword@pulse.mozilla.org:5672/?ssl=1"
+config = env.url("PULSE_DATA_INGESTION_CONFIG", default="")
```

Comment:
iirc the only reason a default was set and then a later `ImproperlyConfigured` behind the conditional, was that we only wanted the error if someone attempted to use pulse ingestion without setting it, but not for general app startup. Now that the setting is no longer in settings.py, we can probably drop the default and avoid the manual `ImproperlyConfigured` raising.

Reply:
Fixed in 998cd8484.  Rebasing didn't make sense since changes to the connection were split across two commits already :)